e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -22,7 +22,7 @@
             "type": "docker",
             "settings": {
               "image": "<Container_Registry>/microsoft/azureiotedge-agent:<Build.BuildNumber>-linux-<Architecture>",
-              "createOptions": ""
+              "createOptions": "{\"HostConfig\":{\"LogConfig\":{\"Type\":\"json-file\",\"Config\":{\"max-size\":\"1m\",\"max-file\":\"3\"}}}}"
```

Comment:
do you know how big the log file size after 1 day of LH test?  I think ideally we should define 7 number of log file with the size of normal log file size for a day.

Reply:
It looks like LoadGen2 from LH output the most log at about 14MB per day. 
While edgeHub log takes about 3.5MB per day, and edgeAgent takes 0.07MB per day

Reply:
That's why I thought the 3MB per module should be about just enough.